### DISCUSSION TITLE
Referal link interferes with website favicon

### DISCUSSION THREAD

**Discussion by Linx-ESP** [JulianPrieber/littlelink-custom#191] [Issues]
Using the docker image, I'm using a referal link for Humble Bundle (https://www.humblebundle.com/?partner=linx_esp) and the link icon is changed to "custom_website".
When loading the littlelink page it shows a generic icon, the console outputs `GET https://icons.duckduckgo.com/ip3/www.humblebundle.com/?partner=linx_esp.ico 404`
https://icons.duckduckgo.com/ip3/www.humblebundle.com/?partner=linx_esp.ico gives a different icon than https://icons.duckduckgo.com/ip3/www.humblebundle.com

I don't how to test if it is because of humble bundle / duckduckgo / littlelink
In case of this issues not being caused by littlelink, is it a way around it? i haven't see an option to manual select a custom icon based on a link (Guess there is an option and didn't see it)

**Comment by JulianPrieber** [comment]
This is more or less a known issue that we've just become used to. The DuckDuckGo API we used to get website icons isn't the best thing. We switched from the Google API, which worked a bit better, and I might add in an option to use this one again.

Right now there inst really a way to get around this. Since the icon you're trying to display is from humblebundle you can use a custom button and get the icon from font awesome.

There are plans to implement buttons with custom icons, but we're still in the planning phase.

**Comment by Linx-ESP** [comment]
Thanks for the answer
